[PROOFREADING] Rundi BTC 101

barakana opened the pull request:
This PR contains the final version of the proofreading work for the document titled "Rundi BTC 101". All identified spelling, grammar, and style improvements have been made to ensure clarity and consistency. Toutes les confusions ont également été résolues, notamment celles liées à certains termes empruntés au kinyarwanda.

Commit:
adding_btc101_rundi

Commit:
adding_rundi_language

barakana committed:
Rundi btc 101 Proofreading Final commit

This is my final version of proofreading that I have just proposed in full.

barakana committed:
Merge branch 'dev' into patch-35

barakana commented:
Hello @MariJJhodl ,
I’m writing to let you know that I’ve finished proofreading, and I wanted to take this opportunity to ask about my next steps. If you see anywhere I could improve, please don’t hesitate to let me know.

Commit:
Update rn.md

MariJJhodl commented:
Last updates required from @barakana:
- write books titles in English, together with some technical words
- correct wrong **

barakana committed:
Update rn.md

I've completed the task.
Checked book titles and technical terms .
Fixed the bold formatting.
Let me know if you need anything else!

barakana commented:
Hi! I’ve wrapped up the work.
Let me know if you need any further tweaks!

Commit:
small fix and paragraph addition
